src/compiler/crystal/program.cr

```diff
@@ -278,33 +278,73 @@ module Crystal
         define_crystal_string_constant "BUILD_COMMIT", build_commit
       else
         define_crystal_nil_constant "BUILD_COMMIT"
-      end
-
-      define_crystal_string_constant "BUILD_DATE", Crystal::Config.date
-      define_crystal_string_constant "CACHE_DIR", CacheDir.instance.dir
-      define_crystal_string_constant "DEFAULT_PATH", Crystal::Config.path
-      define_crystal_string_constant "DESCRIPTION", Crystal::Config.description
-      define_crystal_string_constant "PATH", Crystal::CrystalPath.default_path
-      define_crystal_string_constant "LIBRARY_PATH", Crystal::CrystalLibraryPath.default_path
-      define_crystal_string_constant "LIBRARY_RPATH", Crystal::CrystalLibraryPath.default_rpath
-      define_crystal_string_constant "VERSION", Crystal::Config.version
-      define_crystal_string_constant "LLVM_VERSION", Crystal::Config.llvm_version
-      define_crystal_string_constant "HOST_TRIPLE", Crystal::Config.host_target.to_s
-      define_crystal_string_constant "TARGET_TRIPLE", Crystal::Config.host_target.to_s
-    end
-
-    private def define_crystal_string_constant(name, value)
-      define_crystal_constant name, StringLiteral.new(value).tap(&.set_type(string))
-    end
-
-    private def define_crystal_nil_constant(name)
-      define_crystal_constant name, NilLiteral.new.tap(&.set_type(self.nil))
-    end
-
-    private def define_crystal_constant(name, value)
+      end.doc = <<-MD
```

Comment:
IMO this warrants assigning the constant to a separate variable first before calling `#doc=`

Reply:
There's no need for this since it's not being reused anywhere.